crates/common/validator/src/contribution_and_proof.rs

```diff
@@ -29,3 +37,37 @@ pub struct SignedContributionAndProof {
     pub message: ContributionAndProof,
     pub signature: BLSSignature,
 }
+
+pub fn get_contribution_and_proof(
+    state: &BeaconState,
+    aggregator_index: u64,
+    contribution: SyncCommitteeContribution,
+    private_key: PrivateKey,
+) -> anyhow::Result<ContributionAndProof> {
+    let selection_proof = get_sync_committee_selection_proof(
+        state,
+        contribution.slot,
+        contribution.subcommittee_index,
+        private_key,
+    )?;
+    Ok(ContributionAndProof {
+        aggregator_index,
+        contribution,
+        selection_proof,
+    })
+}
```

Comment:
```suggestion
pub fn get_contribution_and_proof(
    state: &BeaconState,
    aggregator_index: u64,
    contribution: SyncCommitteeContribution,
    private_key: PrivateKey,
) -> anyhow::Result<ContributionAndProof> {
    Ok(ContributionAndProof {
        selection_proof: get_sync_committee_selection_proof(
        state,
        contribution.slot,
        contribution.subcommittee_index,
        private_key,
    )?,
        aggregator_index,
        contribution,
    })
}
```

we should just  inline this all, there isn't a point in declaring a variable for a function this simple, everything might as well be inlined